src/TesterInternal/GeoClusterTests/BasicMultiClusterTest.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Orleans;
+using Orleans.Runtime;
+using Orleans.Runtime.Configuration;
+using System.Net;
+using Orleans.TestingHost;
+
+
+// ReSharper disable InconsistentNaming
+
+namespace Tests.GeoClusterTests
+{
+
+    [TestClass]
+    [DeploymentItem("ClientConfigurationForTesting.xml")]
+    [DeploymentItem("OrleansConfigurationForTesting.xml")]
+    public class BasicMultiClusterTest : TestingClusterHost
+    {
+        [ClassInitialize]
+        public static void ClassInitialize(TestContext testContext)
+        {
+            TestingSiloHost.StopAllSilos();
+        }
+
+        // Kill all clients and silos.
+        [TestCleanup]
+        public void TestCleanup()
+        {
+            try
+            {
+                StopAllClientsAndClusters();
+            }
+            catch (Exception e)
+            {
+                WriteLog("Exception caught in test cleanup function: {0}", e);
+            }
+        }
+
+        // We use ClientWrapper to load a client object in a new app domain. 
+        // This allows us to create multiple clients that are connected to different silos.
+        // this client is used to call into the management grain.
+        public class ClientWrapper : ClientWrapperBase
+        {
+            public ClientWrapper(string name, int gatewayport) : base(name, gatewayport)
+            {
+                systemManagement = GrainClient.GrainFactory.GetGrain<IManagementGrain>(RuntimeInterfaceConstants.SYSTEM_MANAGEMENT_ID);
```

Comment:
this does  not have to be RuntimeInterfaceConstants.SYSTEM_MANAGEMENT_ID. It can be ANY grain. IManagementGrain is pass through, can be any grain instance.